### ISSUE TITLE
Some reports have HTML but no JSON 

### ISSUE THREAD

**Issue by mkhraisha** [w3c-ccg/traceability-interop#208]
Many interop reports have an HTML but no associated JSON which results in them not showing up on the interop reporter
![image](https://user-images.githubusercontent.com/19310771/164759485-3118e722-ead9-4d6b-9ffd-a822698166c2.png)

**Comment by mprorock** [comment]
@brownoxford

**Comment by brownoxford** [comment]
It looks like the presentation exchange is failing to populate the variables needed for the URLs to start and end the exchange when either Transmute or mesur.io is the verifier. I'll nee to dig into this to try and figure out why JSON is not being written in these cases (or if this is even the root cause). When run locally on the command-line, I get both JSON and HTML output for example for the `presentations-verify-mesur.io` test case.

**Comment by brownoxford** [comment]
Confirmed that if a test fails with invalid url, e.g., 
```→ Initiate Exchange
  POST /presentations/available [errored]
     Invalid URI "http:///presentations/available"
```
the JSON file does not get written. If I write a pre-test script with a default value for the exchange endpoint, no error occurs and the JSON file gets written.
It is not clear why this is happening on the test runner, as a local invocation using the same command does write both json and html.
I don't think that using a default base url here is a good solution, will continue to poke at this.

**Comment by brownoxford** [comment]
This has been fixed and merged with #209
